src/Integration.Vsix/Manifests/VS2015_v3Manifest/source.extension.vsixmanifest

```diff
@@ -22,6 +22,7 @@
   </Installation>
   <Dependencies>
     <Dependency Id="Microsoft.Framework.NDP" DisplayName="Microsoft .NET Framework" d:Source="Manual" Version="[4.6,)" />
+    <Dependency d:Source="Project" d:ProjectName="EmbeddedVsix" Version="[1.0]" d:InstallSource="Embed" d:VsixSubPath="EmbeddedVsix" Location="|EmbeddedVsix;VSIXContainerProjectOutputGroup|" DisplayName="|EmbeddedVsix;VSIXNameProjectOutputGroup|" Id="|EmbeddedVsix;VSIXIdentifierProjectOutputGroup|" />
```

Comment:
And this is the magic that embeds one VSIX inside another.

The version is hard-coded at the moment. I'll add a ticket to deal with that later.